mailpoet/lib/Services/AuthorizedSenderDomainController.php

```diff
@@ -10,9 +10,9 @@
 use MailPoet\WP\Functions as WPFunctions;
 
 class AuthorizedSenderDomainController {
-  const OVERALL_STATUS_VERIFIED = 'verified';
-  const OVERALL_STATUS_PARTIALLY_VERIFIED = 'partially-verified';
-  const OVERALL_STATUS_UNVERIFIED = 'unverified';
+  const DOMAIN_STATUS_VERIFIED = 'verified';
```

Comment:
The specs say to `replace OVERALL with DOMAIN_VERIFICATION`, but `DOMAIN` also works, I think.